pkg/handlers/cluster_for_akodeploymentconfig_handler.go

```diff
@@ -32,59 +31,25 @@ func AkoDeploymentConfigForCluster(c client.Client, log logr.Logger) handler.Map
 			return nil
 		}
 		logger := log.WithValues("cluster", cluster.Namespace+"/"+cluster.Name)
-		if SkipCluster(cluster) {
+		if ako_operator.SkipCluster(cluster) {
 			logger.Info("Skipping cluster in handler")
 			return []reconcile.Request{}
 		}
-
-		adcForCluster, err := ListADCsForCluster(ctx, cluster, logger, c)
+		adcForCluster, err := ako_operator.UpdateClusterAKODeploymentConfigLabel(ctx, c, logger, cluster)
```

Comment:
Again, lets avoid such updates from function that is supposed to just return a list of adc's. Ideally the caller should do such updates.